specs/agents/tracing-spans-destination.md

```diff
@@ -100,13 +100,39 @@ exit = exit || context.destination || context.db || context.message || context.h
 
 If no value is set to the `context.destination.service.resource` field, the logic for automatically inferring 
 it MUST be the following:
+
+OPTION I:
 ```groovy
-if      (context.db?.instance)         "${subtype ?: type}/${context.db?.instance}"
+if      (context.db?.instance)         "${subtype ?: type}/${context.db.instance}"
+else if (context.db)                   subtype ?: type
 else if (context.message?.queue?.name) "${subtype ?: type}/${context.message.queue.name}"
-else if (context.http?.url)            "${context.http.url.host}:${context.http.url.port}"
+else if (context.message)              subtype ?: type
+else if (context.http?.url?.port > 0)  "${context.http.url.host}:${context.http.url.port}"
```

Comment:
Maybe :80 and :443 should be omitted

Reply:
While this makes sense, this is not how this field is calculated currently for HTTP spans, which means that an environment that contains a mix of old and new agents will create two nodes for each exit point. So although this makes sense and is doable, I think we better keep it for now.